app/util/files/Report.js

```diff
@@ -0,0 +1,16 @@
+/**
+You can't download a file using a direct AJAX call so use this hidden iframe.
+See comments at https://stackoverflow.com/questions/20499959/extjs-4-downloading-a-file-through-ajax-call and
+https://stackoverflow.com/questions/18434962/extjs-handling-success-or-failure-when-doing-a-standard-submit-in-a-form
+*/
+Ext.define('CpsiMapview.util.files.Report', {
```

Comment:
Could be a view rather than in util folder?

Reply:
I guess that's going to be refactored/replaced sooner or later so I didn't care much, I just put it together with the files that need it, I don't think it's ever going to be used anywhere else

Reply:
Unsure what will be refactored sooner or later?